rasa/nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -83,14 +103,8 @@ def transform_labels_num2str(self, y: np.ndarray) -> np.ndarray:
 
         return self.le.inverse_transform(y)
 
-    def train(
-        self,
-        training_data: TrainingData,
-        config: Optional[RasaNLUModelConfig] = None,
-        **kwargs: Any,
-    ) -> None:
+    def train(self, training_data: TrainingData, **kwargs: Any,) -> Resource:
```

Comment:
let's pass in `num_threads` via the config and drop the `kwargs`. Seems cleaner to me.